Actions/AL-Go-Helper.ps1

```diff
@@ -1914,17 +1914,20 @@ function GetBaseFolder {
     Param(
         [string] $folder
     )
+    
+    Push-Location $folder
+    try {
+        $baseFolder = invoke-git rev-parse --show-toplevel -returnValue
```

Comment:
You can add -silent to avoid that the invoke-git prints out the result on the host.
I am OK with either.